File: cpp/luogu/old/P1271.cpp
/*@author    Allergy
 *@email     Allergy527@gmail.com
 *@workspace luogu\p1271.cpp
 *@data      2024/01/17 19:06:01
 */
#include<bits/stdc++.h>
#define ll long long

using namespace std;

void solve() {
    int n, m;
    cin >> n >> m;
    vector<int>qwq(m);
    for(int i = 0;i < m;++i)cin >> qwq[i];
    sort(qwq.begin(), qwq.end());
    for(auto x : qwq)cout << x << ' ';
}

int main() {
    ios::sync_with_stdio(false);
    int t = 1;
    //cin >> t;
    while(t--)solve();
    cin >> t;
    return 0;
}

File: rust_luogu/src/bin/p1271.rs
//@author    Allergy
//@email     Allergy527@gmail.com
//@workspace luogu/p1271.rs
//@data      2024/01/17 19:03:38
fn cin() -> String {
    let mut input = String::new();
    std::io::stdin().read_line(&mut input).unwrap();
    input.trim().to_string()
}
fn main() {
    let t = 1;
    //let mut input = String::new();std::io::stdin().read_line(&mut input).unwrap();let t:i32=input.parse().unwrap();
    (0..t).for_each(|_| solve());
}
fn solve() {
    cin();
    let mut qwq = cin()
        .split_whitespace()
        .map(|s| s.parse().unwrap())
        .collect::<Vec<i32>>();
    qwq.sort_unstable();
    qwq.iter().for_each(|x| print!("{} ", x))
}
